fleetspeak/src/server/mysql/messagestore.go

```diff
@@ -744,6 +744,7 @@ func (d *Datastore) internalServerMessagesForProcessing(ctx context.Context, lim
 	if err != nil {
 		return nil, err
 	}
+	defer stmt.Close()
```

Comment:
It might be worth adding a comment that this is required here, because we use db.PrepareStatement, but not in other places because we normally prepare statements as part of a transaction, and the transaction then closes them on commit or abort.